mlos_bench/mlos_bench/environments/composite_env.py

```diff
@@ -51,7 +51,13 @@ def __init__(self,
         super().__init__(name=name, config=config, global_config=global_config,
                          tunables=tunables, service=service)
 
-        _LOG.debug("Build composite environment '%s' START: %s", self, self.tunable_params)
+        # By default, the Environment includes only the tunables explicitly specified
+        # in the "tunable_params" section of the config. `CompositeEnv`, however, must
+        # retain all tunables from its children environments plus the ones that come
```

Comment:
```suggestion
        # retain all tunables from its parent environment plus the ones that come
```

?

Reply:
Don't tunables flow down, not up?

Reply:
I guess I'm also not clear on why the CompositeEnv needs to include its own tunables at all.  What is it tuning?  Shouldn't those belong in one of the other Envs? (OSEnv, VMEnv, RemoteEnv, etc.?)

Reply:
Nm, we talked thru this.  I was misunderstanding.